bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/overlay/FindReplaceOverlay.java

```diff
@@ -167,15 +175,48 @@ private void setTextEditorActionsActivated(boolean state) {
 			if (!(targetPart instanceof AbstractTextEditor)) {
 				return;
 			}
+			if (targetPart.getSite() instanceof MultiPageEditorSite multiEditorSite) {
+				if (!state && globalActionHandlerDeaction == null) {
+					globalActionHandlerDeaction = new DeactivateGlobalActionHandlers(multiEditorSite.getActionBars());
+				} else if (state && globalActionHandlerDeaction != null) {
+					globalActionHandlerDeaction.reactivate();
+					globalActionHandlerDeaction = null;
+				}
+			}
 			try {
 				Method method = AbstractTextEditor.class.getDeclaredMethod("setActionActivation", boolean.class); //$NON-NLS-1$
 				method.setAccessible(true);
 				method.invoke(targetPart, Boolean.valueOf(state));
-			} catch (IllegalArgumentException | InvocationTargetException | IllegalAccessException | SecurityException | NoSuchMethodException ex) {
+			} catch (IllegalArgumentException | InvocationTargetException | IllegalAccessException | SecurityException
+					| NoSuchMethodException ex) {
```

Comment:
If suitable you could combine all reflective operations being catched.
```suggestion
			} catch (IllegalArgumentException | ReflectiveOperationException ex) {
```

Reply:
Agreed. Replaced in 5c7a21c